src/applications/personalization/dashboard-2/components/DashboardWrapper.jsx

```diff
@@ -34,14 +35,14 @@ function DashboardWrapper({ showDashboard2, rootUrl }) {
   return <Suspense fallback={loader}>{content}</Suspense>;
 }
 
-const mapStateToProps = state => {
-  const LSDashboardVersion = localStorage.getItem('DASHBOARD_VERSION');
-  const LSDashboard1 = LSDashboardVersion === '1';
-  const LSDashboard2 = LSDashboardVersion === '2';
-  const FFDashboard2 = selectShowDashboard2(state);
-  return {
-    showDashboard2: LSDashboard2 || (FFDashboard2 && !LSDashboard1),
-  };
+DashboardWrapper.propTypes = {
```

Comment:
Added prop types here

Reply:
@erikphansen or @sanlouise, please pull down this branch and try out this component locally (`/dashboard-2/components/DashboardWrapper`) to make sure I didn't break anything when consolidating the selector logic, thank you!! 🙏 🙂